src/Ext/freecad/CMakeLists.txt

```diff
@@ -38,6 +38,9 @@ endif()
 INSTALL(
     FILES
         ${NAMESPACE_INIT}
+        part.py
+        partdesign.py
+        sketcher.py
```

Comment:
should we add one of these for each wb?

Reply:
I understand those are mostly lower-case versions? Indeed that would deserve a discussion... While it is more pythonic, it might also spread different versions of python scripts, and then why not fix the original one instead..

Reply:
more than that I think the idea is that they can get imported directly from python with something like
```python
from freecad import part
```
rather than requiring two lines
```python
import freecad
import Part
```